lisa/microsoft/testsuites/cloud_hypervisor/ch_tests_tool.py

```diff
@@ -358,6 +396,15 @@ def run_metrics_tests(
         skip: Optional[List[str]] = None,
         subtest_timeout: Optional[int] = None,
     ) -> None:
+        # Initialize perf-stable profile if enabled (one-time per metrics test run)
```

Comment:
Mark an environment dirty after the metrics test, so it won't affect results of other test cases.